File: Programmers/Level1/220831/Java/WorkoutClothesTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class WorkoutClothesTest {
  @Test
  void solution() {
    WorkoutClothes workoutClothes = new WorkoutClothes();

    assertEquals(5, workoutClothes.solution(
        5, new int[]{2, 4}, new int[]{1, 3, 5}));
  }

  @Test
  void lostAndReserve() {
    WorkoutClothes workoutClothes = new WorkoutClothes();

    assertEquals(9, workoutClothes.solution(
        10, new int[]{1, 2, 5, 6, 9}, new int[]{1, 3, 5, 6}));
  }

  @Test
  void sort() {
    WorkoutClothes workoutClothes = new WorkoutClothes();

    assertEquals(4, workoutClothes.solution(
        4, new int[]{4, 2}, new int[]{3, 5}));
  }

  @Test
  void exception() {
    WorkoutClothes workoutClothes = new WorkoutClothes();

    assertEquals(4, workoutClothes.solution(
        5, new int[]{1, 2, 4}, new int[]{2, 4, 5}));
  }
}

File: Programmers/Level1/220901/JavaScript/WorkoutClothes.test.js
import { solution } from './WorkoutClothes';

test('solution', () => {
  expect(solution(5, [2, 4], [1, 3, 5])).toBe(5);
  expect(solution(5, [2, 4], [3])).toBe(4);
  expect(solution(3, [3], [1])).toBe(2);
});

test('lostAndReserve', () => {
  expect(solution(10, [1, 2, 5, 6, 9], [1, 3, 5, 6])).toBe(9);
  expect(solution(3, [1, 2], [2, 3])).toBe(2);
});
